cluster/gce/win1803/configure.ps1

```diff
@@ -0,0 +1,119 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
nit: change all new files to use `Copyright 2019`

Reply:
I was wondering about that. Done.